config/samples/node_example_loadtest_with_pre_built_workers.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  # Every load test instance must be assigned a unique name on the
+  # cluster. There are ways we can circumvent naming clashes, such
+  # as using namespaces or dynamically assigning names.
+  name: prebuilt-node-example
+
+  # As a custom resource, it behaves like a native kubernetes object.
+  # This means that users can perform CRUD operations through the
+  # Kubernetes API or kubectl. In addition, it means that the user
+  # can set any metadata on it.
+  labels:
+    language: node
+spec:
+  # The user can specify servers to use when running tests. The
+  # initial version only supports 1 server to limit scope. Servers
+  # is an array for future expansion.
+  #
+  # There are many designs and systems to pursue load balancing,
+  # organizing and monitoring a mesh of servers. Therefore, this
+  # will likely be expanded in the future.
+  servers:
+  - language: node
+    run:
+      image: gcr.io/grpc-testing/donghao/prebuilt/node
```

Comment:
I think we could leave it as 'image: <current_image_repository>' or something else,  since we will use tools/run_tests/performance/loadtest_template.py to alter the field into a variable that could be changed in https://github.com/grpc/grpc/blob/master/tools/run_tests/performance/loadtest_config.py. The examples employing prebuilt images could be served as template to generate the basic template. Let me know if you think this makes scene. :) @codeblooded, do you think this make scene?

Reply:
This has been addressed: configuration is now a template, and the variables in it will be passed unchanged to the mutli-language template.